README.md

```diff
@@ -576,4 +576,4 @@ In the meantime, there are two complementary and simultaneous activities going o
 
 You can keep track of the latest developments by filtering all the PRs according to the type of activity:
 
-[![NativeBase's dismissal label](https://img.shields.io/github/labels/pagopa/io-app/NativeBase%20dismissal%20%F0%9F%A5%B7%F0%9F%8F%BC)](https://github.com/pagopa/io-app/labels/NativeBase%20dismissal%20%F0%9F%A5%B7%F0%9F%8F%BC) [![Design System's label](https://img.shields.io/github/labels/pagopa/io-app/Design%20System)](https://github.com/pagopa/io-app/labels/Design%20System)
\ No newline at end of file
+[![NativeBase's dismissal label](https://img.shields.io/github/labels/pagopa/io-app/NativeBase%20dismissal%20%F0%9F%A5%B7%F0%9F%8F%BC)](https://github.com/pagopa/io-app/labels/NativeBase%20dismissal%20%F0%9F%A5%B7%F0%9F%8F%BC) [![Design System's label](https://img.shields.io/github/labels/pagopa/io-app/Design%20System)](https://github.com/pagopa/io-app/labels/Design%20System)
```

Comment:
Why did this line change?

Reply:
Probably different end of line management

Reply:
Checked with difference checker but nothing is changed in this line.

Reply:
Hi @omimakhare, please discard such hunk in order for us to approve your PR